plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java

```diff
@@ -740,6 +743,44 @@ public void testPartitionFilterRequired()
         assertUpdate(session, "DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testTableChangesOnMultiRowGroups()
```

Comment:
This test should go in `TestIcebergParquetConnectorTest`, base class tests are supposed to be file format agnostic